tasks/task721_hate_speech18_classification.json

```diff
@@ -0,0 +1,1669 @@
+{
+    "Contributors": [
+        "Nisarg Patel"
+    ],
+    "Source": [
+        "Hate Speech Recognition (https://huggingface.co/datasets/hate_speech18)"
+    ],
+    "Categories": [
+        "Classification"
+    ],
+    "Definition": "Given a statement, determine if the sentiment is of hatred, no hate,relation or nothing at all.",
```

Comment:
Please explain the numbers in the input. Anyone should be able to do the task by just reading the definition. You can ask one of your friends to read the definition and solve one of the instances.
Also, consider adding a space after comma in no hate,.